src/app/parser.rs

```diff
@@ -1533,6 +1533,11 @@ impl<'a, 'b> Parser<'a, 'b>
                 return Err(Error::value_validation(Some(arg), e, self.color()));
             }
         }
+        if let Some(vtor) = arg.validator_os() {
+            if let Err(e) = vtor(val.to_string_lossy().into_owned()) {
```

Comment:
Once the `String`->`OsString` is done, this should be able to change to just `vtor(val)`